citi-onboarding/Aqua-Jr
# Hotfix/testimony front

# What i did:
- Fix the responsivity of the Testimony and Newsletter
- Fix the css of the Testimony and Newsletter
# How to test:
1. `git checkout develop`
2. `git pull`
3. `git checkout hotfix/Testimony-front`
4. See the size of the fonts, see the responsivity and see if the button has a method pointer
